pkgs/development/python-modules/unidic/default.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, mecab
+, setuptools-scm
+, requests
+, tqdm
+, wasabi
+, plac
+, cython
+}:
+
+buildPythonPackage rec {
+  pname = "unidic";
+  version = "1.1.0";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "polm";
+    repo = "unidic-py";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-srhQDXGgoIMhYuCbyQB3kF4LrODnoOqLbjBQMvhPieY=";
+  };
+
+  postPatch = ''
```

Comment:
`python -m unidic download` is failed.
It seems to need to explicitly fetch the dict and patch the `dicdir`.